smithy-aws-iam-traits/src/test/java/software/amazon/smithy/aws/iam/traits/ConditionKeysIndexTest.java

```diff
@@ -65,4 +60,49 @@ public void successfullyLoadsConditionKeys() {
         assertThat(index.getDefinedConditionKeys(service, ShapeId.from("smithy.example#GetResource2")).keySet(),
                 is(empty()));
     }
+
+    @Test
```

Comment:
Can these tests validate that the `getConditionKeyNames` contain keys specified via the `conditionKeys` trait?